### ISSUE TITLE
Reparenting and nested grids

### ISSUE THREAD

**Issue by k4st0r42** [haltu/muuri#387]
I want to have hierarchy in my grid system.

I have identified 2 issues for that :

When I want to drag and drop from a parent grid to a grid in a child element, the child move (due to the collision algorithm I think ?). So, I can't drop in the grid in the child element...

When I want to drag and drop from a grid in child element to itself, this is dropped to the parent grid.
In this case, I think it should be necessary to priorize the child grid to drop.

Video record :
https://drive.google.com/open?id=1Xuo1VPuobObr13CLDlIne9XMuTj05yvl
 
Related to #95 
Any solution to this problem ?

Thank you

**Comment by niklasramo** [comment]
Yes, you need to create a custom [`dragSortPredicate`](https://github.com/haltu/muuri#dragsortpredicate-). The reason the default predicate does not handle this kind of scenario is that it's really hard to come up with an API that supports both scenarios (dragging into parent grid or some descendant grid, or maybe both). Which is why I left the API open for a custom solution.

**Comment by k4st0r42** [comment]
I will investigate
Thank you for the advice

**Comment by luizottavioc** [comment]
@LudovicF42 did you find a solution to resolve this? I'm having the same problem here.
